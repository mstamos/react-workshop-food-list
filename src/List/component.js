
function List({items}) {
    return (
      <ul>
        {items
          .map(item => {
            return (
            <li key={item}>{item}</li> 
            )
          })
         }
      </ul>
    );
  }
  

export default List;