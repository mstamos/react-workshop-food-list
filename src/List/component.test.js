import {render} from '@testing-library/react';
import List from './component';

describe('<List/>', () => {
    it('renders', () => {
        expect(render(<List items={['spanakorizo', 'burger']}/>).container).toMatchSnapshot();
    })
})