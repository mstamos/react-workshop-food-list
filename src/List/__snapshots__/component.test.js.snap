// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<List/> renders 1`] = `
<div>
  <ul>
    <li>
      spanakorizo
    </li>
    <li>
      burger
    </li>
  </ul>
</div>
`;
